app/models/consent_form.rb

```diff
@@ -436,6 +437,8 @@ def parent_relationship
   end
 
   def match_with_patient!(patient, current_user:)
+    send_parental_contact_warning_if_needed(patient, self, current_user)
```

Comment:
Actually I'm not sure if this is the right place to put this call. The concern is a controller concern, so already it feels wrong to include it in the model. I think it would be better to call this wherever `match_with_patient!` is called, and in fact I think that fits better with checking for the user as then we only need to call it from the automatic matching job.